templates/hpa.yaml

```diff
@@ -1,5 +1,11 @@
 {{- if .Values.autoscaling.enabled }}
+{{- if semverCompare ">=1.23-0" .Capabilities.KubeVersion.GitVersion -}}
 apiVersion: autoscaling/v2
+{{- else if semverCompare ">=1.12-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: autoscaling/v2beta2
+{{- else -}}
+apiVersion: autoscaling/v1beta1
+{{- end }}
```

Comment:
I can't seem to find any reference to an autoscaling version `v1beta1`. I do see a `v1` though.

Should this be `v1` or, if there is a `v1beta1`, can you point me in the right direction of the docs?

Reply:
Hey, oh you're right, mistake on my side 🤦🏻 this one should be `v1`